invenio_oauthclient/alembic/97bbc733896c_create_oauthclient_tables.py

```diff
@@ -73,6 +73,7 @@ def upgrade():
         sa.Column('secret', sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
             ['id_remote_account'], [u'oauthclient_remoteaccount.id'],
+            name='fk_oauthclient_remote_token_remote_account'
```

Comment:
I understand @jirikuncar's point and if the package were "fully" released I would agreed with him, but given the fact that the package is in `alpha` stage I would just update the recipe as @nikofil did.